onnxscript/function_libs/torch_aten/ops/nn.py

```diff
@@ -198,10 +198,11 @@ def aten_binary_cross_entropy_backward(
     raise NotImplementedError()
 
 
-def aten_celu(self, alpha: float = 1.0):
+@torch_op("aten::celu")
+def aten_celu(self: TFloat, alpha: float = 1.0) -> TFloat:
     # celu(Tensor self, Scalar alpha=1.0) -> Tensor
 
-    raise NotImplementedError()
+    return op.Celu(self, alpha=alpha)
```

Comment:
The input/output of Celu is limited to float32, so we might need a dtype to cast the value back.

Reply:
We don't really know what the input dtype is. I think this can be handled by the caller.